GeoMessageSpecification/GeoMessageSpecification.md

```diff
@@ -0,0 +1,90 @@
+GeoMessage Specification
```

Comment:
Minor (no need to fix, just something to keep in mind) - it might be good if this file were called Readme.md instead so it would be auto-displayed when view from this folder on Github